x-pack/solutions/security/plugins/security_solution/server/endpoint/services/workflow_insights/helpers.ts

```diff
@@ -136,16 +136,23 @@ export async function groupEndpointIdsByOS(
 }
 
 export function generateInsightId(insight: SecurityWorkflowInsight): string {
-  const { type, category, value, target } = insight;
+  const { type, category, value, target, remediation } = insight;
   const targetType = target.type;
   const targetIds = target.ids.join(',');
+  const targetRemediationEntries =
```

Comment:
I feel a bit iffy about adding `remediation.exception_list_items.entries.value` into the hashed id. `exception_list_items` is optional as there will likely be future insight types where this is not provided. I think we should just make the top level `insight.value` a signer+path combo since that's the intent of the `insight.value` field.